src/libse/SubtitleFormats/PacUnicode.cs

```diff
@@ -142,13 +142,29 @@ private static Paragraph GetPacParagraph(ref int index, byte[] buffer)
                         {
                             if (buffer[j] == 0xff)
                             {
+                                // I never hit this on a breakpoint...
                                 buffer[j] = 0x2e; // replace end of line marker
                             }
                         }
-
+                        
                         sb.AppendLine(Encoding.UTF8.GetString(buffer, textBegin, textIndex - textBegin));
-                        textBegin = textIndex + 7;
-                        textIndex += 6;
+                        for (var v = 0; v < 5; v++)
+                        {
+                            if (buffer[textIndex + 1 + v] == 0x1F &&
+                                buffer[textIndex + 2 + v] == 0xEF &&
+                                buffer[textIndex + 3 + v] == 0xBB &&
+                                buffer[textIndex + 4 + v] == 0xBF)
+                            {
+                                // skip utf8 BOM
+                                textBegin = textIndex + 7;
+                                textIndex += 6;
+                            }
+                            else
+                            {
+                                // ASCI or Header Paragraph no BOM 
+                                textBegin = textIndex + 1;
+                            }
+                        }
```

Comment:
This checks and resolves an issue with the "Header" Paragraphs using ASCI which was skipping content, and no BOM, while keeping the BOM skip in, stopping double line breaks later in the decode / display. 

Header Packets written from Subtitle Edit won't have this issues, as every paragraph is treated the same and the BOM is applied to every line.